src/Sentry.Unity.Editor/UnityCommandLineArguments.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+
+namespace Sentry.Unity.Editor
+{
+    internal static class UnityCommandLineArguments
```

Comment:
Not really "Unity" specific, and what it does is parsing CLI args so what about:

```suggestion
    internal static class CommandLineArgumentParser
```